File: 001-BitwiseOperators/017.number-is-sparse-or-not.js
const isSparse = (n) => {
  let prev = 0;
  while (n) {
    let t = n % 2;
    if (prev == 1 && t == 1) {
      return false;
    }
    prev = t;
    n = parseInt(n / 2);
  }
  return true;
};

File: Geeks-For-Geeks/001-BitwiseOperators/017.number-is-sparse-or-not.py
class Solution:
    
    #Function to check if the number is sparse or not.
    def isSparse(self,n):
        prev = 0
        while n:
            t = n % 2
            if prev == 1 and t == 1:
                return False
            prev = t
            n = n // 2
        
        return True